Rivet/Functions/Update-Database.ps1

```diff
@@ -133,38 +131,31 @@ function Update-Database
         $byName['Include'] = $Name
     }
 
-    Get-MigrationFile -Path $Path -Configuration $Configuration @byName -ErrorAction Stop |
+    $query = 'if (object_id(''{0}'', ''U'') is not null) select ID, Name, Who, AtUtc from {0}' -f $RivetMigrationsTableFullName

```

Comment:
Specific warning if missing Rivet tables? Or is that elsewhere?

Reply:
This is for first-run situation where Rivet is being run for the first time so not only are there no migrations, but there are no Rivet objects in the database.